<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>760</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>ISAT</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/icons/isat_bg_50x25.png</normaloff>:/icons/icons/isat_bg_50x25.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_dir"/>
    <addaction name="actionSave_dir"/>
    <addaction name="separator"/>
    <addaction name="actionPiror"/>
    <addaction name="actionNext"/>
    <addaction name="separator"/>
    <addaction name="actionSetting"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionZoom_in"/>
    <addaction name="actionZoom_out"/>
    <addaction name="actionFit_wiondow"/>
    <addaction name="separator"/>
    <addaction name="actionBit_map"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <widget class="QMenu" name="menuLaguage">
     <property name="title">
      <string>Laguage</string>
     </property>
     <property name="icon">
      <iconset resource="../icons.qrc">
       <normaloff>:/icons/icons/翻译_translate.svg</normaloff>:/icons/icons/翻译_translate.svg</iconset>
     </property>
     <addaction name="actionChinese"/>
     <addaction name="actionEnglish"/>
    </widget>
    <addaction name="menuLaguage"/>
    <addaction name="actionShortcut"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionConverter"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCreate"/>
    <addaction name="actionEdit"/>
    <addaction name="actionDelete"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionTo_top"/>
    <addaction name="actionTo_bottom"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_dir"/>
   <addaction name="actionSave_dir"/>
   <addaction name="actionPiror"/>
   <addaction name="actionNext"/>
   <addaction name="actionBit_map"/>
   <addaction name="separator"/>
   <addaction name="actionCreate"/>
   <addaction name="actionEdit"/>
   <addaction name="actionDelete"/>
   <addaction name="actionSave"/>
   <addaction name="actionTo_top"/>
   <addaction name="actionTo_bottom"/>
   <addaction name="separator"/>
   <addaction name="actionFit_wiondow"/>
   <addaction name="actionZoom_in"/>
   <addaction name="actionZoom_out"/>
  </widget>
  <widget class="QDockWidget" name="info_dock">
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>38</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Info</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
  <widget class="QDockWidget" name="labels_dock">
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>38</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Labels</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3"/>
  </widget>
  <widget class="QDockWidget" name="files_dock">
   <property name="windowTitle">
    <string>Files</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <action name="actionOpen_dir">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/照片_pic.svg</normaloff>:/icons/icons/照片_pic.svg</iconset>
   </property>
   <property name="text">
    <string>Images dir</string>
   </property>
   <property name="statusTip">
    <string>Open images dir.</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/放大_zoom-in.svg</normaloff>:/icons/icons/放大_zoom-in.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="statusTip">
    <string>Zoom in.</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/缩小_zoom-out.svg</normaloff>:/icons/icons/缩小_zoom-out.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="statusTip">
    <string>Zoom out.</string>
   </property>
  </action>
  <action name="actionFit_wiondow">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/等比_equal-ratio.svg</normaloff>:/icons/icons/等比_equal-ratio.svg</iconset>
   </property>
   <property name="text">
    <string>Fit window</string>
   </property>
   <property name="toolTip">
    <string>Fit window</string>
   </property>
   <property name="statusTip">
    <string>Fit window.</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/设置_setting-two.svg</normaloff>:/icons/icons/设置_setting-two.svg</iconset>
   </property>
   <property name="text">
    <string>Setting</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/开关_power.svg</normaloff>:/icons/icons/开关_power.svg</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSave_dir">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/文件夹-开_folder-open.svg</normaloff>:/icons/icons/文件夹-开_folder-open.svg</iconset>
   </property>
   <property name="text">
    <string>Label dir</string>
   </property>
   <property name="statusTip">
    <string>Open label dir.</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/保存_save.svg</normaloff>:/icons/icons/保存_save.svg</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="statusTip">
    <string>Save annotation.</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actionPiror">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/上一步_back.svg</normaloff>:/icons/icons/上一步_back.svg</iconset>
   </property>
   <property name="text">
    <string>Priro image</string>
   </property>
   <property name="toolTip">
    <string>Prev image</string>
   </property>
   <property name="statusTip">
    <string>Prior image.</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
   <property name="priority">
    <enum>QAction::NormalPriority</enum>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/下一步_next.svg</normaloff>:/icons/icons/下一步_next.svg</iconset>
   </property>
   <property name="text">
    <string>Next image</string>
   </property>
   <property name="toolTip">
    <string>Next image</string>
   </property>
   <property name="statusTip">
    <string>Next image.</string>
   </property>
   <property name="shortcut">
    <string>D</string>
   </property>
  </action>
  <action name="actionShortcut">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/键盘_keyboard-one.svg</normaloff>:/icons/icons/键盘_keyboard-one.svg</iconset>
   </property>
   <property name="text">
    <string>Shortcut</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/头像_avatar.svg</normaloff>:/icons/icons/头像_avatar.svg</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionCreate">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/编辑撰写_writing-fluently.svg</normaloff>:/icons/icons/编辑撰写_writing-fluently.svg</iconset>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
   <property name="toolTip">
    <string>Create polygon</string>
   </property>
   <property name="statusTip">
    <string>Draw polygon.</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/删除_delete.svg</normaloff>:/icons/icons/删除_delete.svg</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete polygon</string>
   </property>
   <property name="statusTip">
    <string>Delete polygon.</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionBit_map">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/照片_pic.svg</normaloff>:/icons/icons/照片_pic.svg</iconset>
   </property>
   <property name="text">
    <string>Bit map</string>
   </property>
   <property name="statusTip">
    <string>Show instance or segmeent state.</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/编辑文件_file-editing.svg</normaloff>:/icons/icons/编辑文件_file-editing.svg</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="toolTip">
    <string>Edit polygon</string>
   </property>
   <property name="statusTip">
    <string>Edit polygon attribute.</string>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
  </action>
  <action name="actionTo_top">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/去顶部_to-top.svg</normaloff>:/icons/icons/去顶部_to-top.svg</iconset>
   </property>
   <property name="text">
    <string>To top</string>
   </property>
   <property name="toolTip">
    <string>Move polygon to top layer</string>
   </property>
   <property name="statusTip">
    <string>Move polygon to top layer.</string>
   </property>
   <property name="shortcut">
    <string>T</string>
   </property>
  </action>
  <action name="actionTo_bottom">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/去底部_to-bottom.svg</normaloff>:/icons/icons/去底部_to-bottom.svg</iconset>
   </property>
   <property name="text">
    <string>To bottom</string>
   </property>
   <property name="toolTip">
    <string>Move polygon to bottom layer</string>
   </property>
   <property name="statusTip">
    <string>Move polygon to bottom layer.</string>
   </property>
   <property name="shortcut">
    <string>B</string>
   </property>
  </action>
  <action name="actionConverter">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/icons/转换文件1_file-conversion-one.svg</normaloff>:/icons/icons/转换文件1_file-conversion-one.svg</iconset>
   </property>
   <property name="text">
    <string>Label converter</string>
   </property>
   <property name="statusTip">
    <string>Convert annotations to png image.</string>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
  </action>
  <action name="actionChinese">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>中文</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
